examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>I realise the semi-colon is a compact way of writing code, but would it be clearer to readers to add the extra 3 lines explcitly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
No problem with me!